learning_paths/api/v1/tests/test_views.py

```diff
@@ -285,13 +285,12 @@ def test_enrollment_returns_404_for_invalid_user_or_learning_path(self):
         self.client.force_authenticate(user=self.staff)
 
         # Test invalid username
-        response = self.client.post(self.url, {"username": "non-existant-user"})
+        response = self.client.post(self.url, {"username": "non-existent-user"})
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
         # Test invalid learning_path_id
-        response = self.client.post(
-            "/api/learning-paths/2ac8a3cc-e492-4ce9-88a3-cce4922ce9df/enrollments/"
-        )  # Invalid Learning Path ID
+        url = reverse("learning-path-enrollments", args=['2ac8a3cc-e492-4ce9-88a3-cce4922ce9df'])
+        response = self.client.post(url)
```

Comment:
Generally, testing API paths can be valuable (to avoid accidental URL changes), but in this case, we were getting a 404 because the URL was invalid. I replaced it with `reverse`, as it's more reliable for this use case.

Reply:
Good tip. Will keep in mind next time.